jsk_pcl_ros/cfg/HintedPlaneDetector.cfg

```diff
@@ -9,6 +9,7 @@ from math import pi
 
 gen = ParameterGenerator ()
 
+gen.add('synchronize', bool_t, 0, 'Enable synchronization of input topics', True)
```

Comment:
Could you update the `hinted_plane_detector.md` doc too?

Reply:
Sure. Updated!